src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -2784,6 +2871,322 @@ static void Test()
             verifier.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void TestSynthesizedConstructorWithSpansInMultipleFilesCoverage()
```

Comment:
>TestSynthesizedConstructorWithSpansInMultipleFilesCoverage [](start = 20, length = 58)

I think @jcouv added an infrastructure to make instrumentation tests more human readable and easier to make sense. Please talk to him to see if it can be used for tests that you are adding.